assets/js/renderer.js

```
@@ -1,186 +1,179 @@
 const generatePublicKey = async () => {
-    privateKey = document.getElementById('private-key-text-area').value
-    let passphrase = document.getElementById('passphrase-input').value || undefined;
-    let publicKey = await window.utils.generatePublicKey(privateKey, passphrase)
-    document.getElementById('public-key-text-area').value = publicKey
-
-    if (publicKey) {
-        change_buttons_disabled_state(false)
-    } else {
-        change_buttons_disabled_state(true)
-    }
+  privateKey = document.getElementById('private-key-text-area').value
```

ESLint

[error] 'privateKey' is not defined.

(no-undef)